test/test_namedtuple_return_api.py

```diff
@@ -11,7 +11,7 @@
 aten_native_yaml = os.path.join(path, '../aten/src/ATen/native/native_functions.yaml')
 all_operators_with_namedtuple_return = {
     'max', 'min', 'median', 'nanmedian', 'mode', 'kthvalue', 'svd', 'symeig', 'eig',
-    'qr', 'geqrf', 'solve', 'slogdet', 'sort', 'topk', 'lstsq',
+    'qr', 'geqrf', 'solve', 'slogdet', 'sort', 'msort', 'topk', 'lstsq',
```

Comment:
msort will be removed from this list once it returns only the tensor.

Reply:
Removed!